src/pages/bridge/Issue/index.tsx

```diff
@@ -78,8 +80,15 @@ function Issue(props: IssueProps): JSX.Element {
           >
             here
           </a>
+          .
+        </li>
+        <li>
+          Estimated time for issuing: In a minute after submitting the Stellar payment to the vault, contact
```

Comment:
How about we end the sentence here. 
```suggestion
          Estimated time for issuing: In a minute after submitting the Stellar payment to the vault. Contact
```

Reply:
Agree